src/components/Project/Paid.tsx

```diff
@@ -161,12 +159,13 @@ export default function Paid() {
           justifyContent: 'space-between',
           alignItems: 'baseline',
           flexWrap: 'nowrap',
+          marginBottom: 15,
```

Comment:
mixed feelings about this. the uniform spacing does seem nice visually, but i think the closeness between `in juicebox` and `distributed` also helps indicate that the two are tied together conceptually

Reply:
Yeah that's probably a good idea. I'll revert